qualtran/cirq_interop/_cirq_interop.py

```diff
@@ -13,11 +13,11 @@
 #  limitations under the License.
 
 """Functionality for the `Bloq.as_cirq_op(...)` protocol"""
-
 import itertools
 from functools import cached_property
 from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Sequence, Tuple, Union
 
+import attr
```

Comment:
annoying nit: can you use attrs instead of attr. It's their cleaned-up API (but the other one will be around forever)

Reply:
Cirq can't use attrs because of a dependency issue due to external vendors. So cirq-ft defaults to using `attr` and so I end up defaulting to using `attr`. I think it should be fine to using `attrs` here since we only depend upon cirq-ft & cirq-core and don't pin attr version dependency. 

Tl;Dr - I'll change it to `attrs`.

Reply:
interesting. I thought installing attr or attrs would get you both `import attr` and `import attrs`

Reply:
`attrs` was introduced in a specific version of `attr` and one of the cirq vendors has their `attr` dependency pinned to an older version so the `attrs` namespace does not exist.